CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributor Guidelines:
+
+All contributions to this project should follow the [Nesta Github Guidelines](https://github.com/nestauk/github_support/blob/dev/guidelines/README.md)
+
+To add a feature:
+
+1. Add an issue that describes the feature
```

Comment:
I think we should encourage people to start a conversation (here or on Slack?) about the issue/feature before going ahead and developing something. It might already exist, or there might be a reason it doesn't, or it might non-obviously be a bad idea.

Reply:
Yeah - maybe these guidelines should start from after a feature has already been proposed and approved for working on. I will create separate instructions for suggesting a feature.